Update error handling page (#355)
saleor/saleor-docs: docs/developer/api-conventions/error-handling.mdx

Although all error types contain a `message` field, the returned message is only meant for debugging and is not suitable for display to your customers. Please use the `code` field and provide your own meaningful error messages.
:::

> For representing code values, we are using enums. All possible values for the query above can be found on [AccountError](developer/api-reference/enums/account-error-code.mdx) reference page.

Reviewer: ```suggestion
We use enums for representing code values. All possible values for the query above can be found on the [AccountError](developer/api-reference/enums/account-error-code.mdx) page of the API reference section.
```
